index.bs

```diff
@@ -10035,16 +10036,16 @@ Constructors</h5>
 	: <dfn>port</dfn>
 	::
 		Every {{AudioWorkletProcessor}} has an associated
-		<code>port</code> which is a {{MessagePort}}. It is connected to the port on the
+		<code>port</code> which is a {{MessagePort}}. It is connected to a port on the
 		corresponding {{AudioWorkletProcessor}} object allowing
```

Comment:
This is the port for the AudioWorkletProcessor.  Should this say it's connected to the port for associated AudioWorkletNode?

Reply:
This port is created by MessageChannel. So the association between port1 and port2 are already implied.

Reply:
Should I undo this change?

Reply:
`the port` is more specific. So yes, I think it's fine as-is.

Reply:
I prefer "the port" because it's more specific.

Reply:
Done.

Reply:
done